docs/CHANGELOG.rst

```diff
@@ -5,6 +5,15 @@ Change Log
 All notable changes to this project are documented in this file.
 
 
+==========
+Unreleased
+==========
+
+Added
+-----
+- Add ``permissions.copy_from()`` method
```

Comment:
I would name it `<resource>.permissions.copy_from(<source>)` and add a short description of what the method does.

Reply:
Good suggestion, but I've modified it a bit.